integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/intygproxyservice/person/webcert/WebcertCertificateIntegrationService.java

```diff
@@ -0,0 +1,9 @@
+package se.inera.intyg.minaintyg.integration.intygproxyservice.person.webcert;
```

Comment:
Change the package to something like: se.inera.intyg.minaintyg.integration.webcert (exact package can be adjusted in future pr:s when this is implemented.